http/templates/season_standings.html.tera

```diff
@@ -6,52 +6,55 @@
 {% block season_body %}
 <div id="standings-page-container">
     {% for bracket in brackets %}
-        <div class="bracket-standings-container">
-            <h2 class="bracket-title">Bracket: {{ bracket.name }}</h2>
-            <table class="bracket-standings">
-                <thead>
-                    <tr>
-                        <td>
-                            Rank
-                        </td>
-                        <td>
-                            Player Name
-                        </td>
-                        <td>
-                            Points
-                        </td>
-                        <td>
-                            Opponent Points
-                        </td>
-                        <td>
-                            Average Time
-                        </td>
-                    </tr>
-                </thead>
-                <tbody>
-                    {% for player in bracket.players %}
-                        <tr>
-                            <td>
-                                {{ loop.index }}
-                            </td>
-                            <td>
-                                {{ macros::link(href=player.player_detail_url, text=player.name) }}
-                            </td>
-                            <td>
-                                {{ player.points }}
-                            </td>
-                            <td>
-                                {{ player.opponent_points }}
-                            </td>
-                            <td>
-                                {{ macros::get_text_or_placeholder(text = player.average_time) }}
-                            </td>
-                        </tr>
-                    {% endfor %}
-                </tbody>
-            </table>
-        </div>
-        <hr class="template-item-separator" />
+    <div class="bracket-standings-container">
+        <h2 class="bracket-title">Bracket: {{ bracket.name }}</h2>
+        <table class="bracket-standings">
+            <thead>
+                <tr>
+                    <td>
+                        Rank
+                    </td>
+                    <td>
+                        Player Name
+                    </td>
+                    <td>
+                        Points
+                    </td>
+                    <td>
+                        Opponent Points
+                    </td>
+                    <td>
+                        Average Time <span class="without-forfeits">(without forfeits)</span>
```

Comment:
Would it be better to throw this time in a new column? Or is that too much space for another time?

Another option could be to make it toggle-able with a button on top or something similar

Reply:
i had it in a separate column at first but i thought it looked bad. a toggle would also be good, but i think this is fine as is, and doing a toggle is extra work 😛

Reply:
Yep that's fair. You can give that work to the intern you plan to hire next summer :3